### PR TITLE
refactor: do not implicitly use nil entity as admin default

### PR DESCRIPTION
**Pull request by jackstar12**

### CODE REVIEW

File: autoswap/autoswap_test.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package autoswap
 
 import (
+	"fmt"
+	rand2 "golang.org/x/exp/rand"
```
Comment (michael1011) [inline autoswap/autoswap_test.go:5]: Why the alias?

File: boltzrpc/boltzrpc.proto [hunk 2]
```diff
@@ -258,6 +258,7 @@ message SwapInfo {
     optional uint64 onchain_fee = 18;
     // internal wallet which was used to pay the swap
     optional int64 wallet_id = 20;
+    int64 entity_id = 21;
```
Comment (michael1011) [inline boltzrpc/boltzrpc.proto:261]: Those ids can't be negative, can they? Why not use `uint64`?

File: database/entity.go [hunk 3]
```diff
@@ -9,6 +9,19 @@ type Entity struct {
 	Name string
 }
 
+const DefaultEntityId Id = 1
+const DefaultEntityName string = "default"
```
Comment (michael1011) [inline database/entity.go:13]: Should we call it admin?

Comment (jackstar12) [reply database/entity.go]: makes sense

**Review by michael1011 — APPROVED** [review]
utACK

**Pull request merged**